README.md

```diff
@@ -41,22 +41,30 @@ In the wake of the Empire's fall, the Republic has turned to Coruscant Bank to a
 
 ## Requirements
 
-Your task is to design the foundation of a user interface capable of identifying planets susceptible to uprisings and enabling proactive intervention. The objective is to build a UI that assists administrators in analyzing transactions across various planets. To meet this goal, adhere to the following guidelines:
+Your task is to design the foundation of a responsive user interface capable of identifying planets susceptible to uprisings and enabling proactive intervention. The objective is to build a UI that assists administrators in analyzing transactions across various planets. To meet this goal, adhere to the following guidelines:
```

Comment:
I like it! It's not as clear as a bullet point, but it shouldn't need to be, just a small reminder.